packages/coin-support-utils/tests/02.createAccount/04.generateAddressesPerScheme/index.ts

```diff
@@ -0,0 +1,21 @@
+import { describe, test } from '@jest/globals';
+
+import { fixtures } from './__fixtures__';
+import { TestApp, getAddressesFromDeviceMock } from '../../__mocks__';
+import {
+  IGenerateAddressesPerSchemeParams,
+  generateAddressesPerScheme,
+} from '../../../src';
+
+describe('04. Generate Addresses Per Scheme', () => {
+  fixtures.valid.forEach(({ name, input, output }) => {
+    test(name, async () => {
+      const params: IGenerateAddressesPerSchemeParams<TestApp> = {
+        derivationPathsPerScheme: input.derivationPathsPerScheme,
+        getAddressesFromDevice: getAddressesFromDeviceMock,
```

Comment:
Add a check to make sure `getAddressesFromDevice` function receives correct parameters.